dev/plugins/hu.elte.txtuml.api.stdlib/src/hu/elte/txtuml/api/stdlib/math/Math.xtxtuml

```diff
@@ -0,0 +1,30 @@
+package hu.elte.txtuml.api.stdlib.math;
+
+/**
+ * Proxy class for the {@linkplain java.lang.Math} class.
+ */
+class Math {
+
+	/**
+	 * This class cannot be instantiated from the model.
+	 */
+	private external-body Math() {}
```

Comment:
I wanted to emphasize the fact that these types are only interfaces above some external features in the model and therefore their constructors are present in the model only to prohibit their instantiation. The body of these constructors (if any) has no part in the model.